test/primitives.jl

```diff
@@ -431,4 +431,10 @@
     @test !isconvex(s)
     @test isnothing(boundary(s))
   end
+
+  @testset "Torus" begin
+    t = Torus(T.((1,1,1)), T.((1,0,0)), 2, 1)
+    @test P3(1,1,-1) ∈ t
+    @test P3(1,1,1) ∉ t
+  end
```

Comment:
Need tests for the accessor functions

Reply:
What are the accessor functions? The radii, the center, etc?

Reply:
Exactly, the functions that return properties of the concept.